packages/core/test/event.test.ts

```diff
@@ -335,4 +335,78 @@ describe('@bugsnag/core/event', () => {
       ])
     })
   })
+
+  describe('Event.create()', () => {
+    it('includes causes in the exceptions array', () => {
+      const err = new Error('I am the error')
+      // @ts-ignore
+      err.cause = new Error('I am the cause')
+      // @ts-ignore
+      const event = Event.create(err, true, undefined, 'notify()', 0)
+      expect(event.errors.length).toBe(2)
+      expect(event.errors).toContainEqual(
+        expect.objectContaining({
+          errorClass: 'Error',
+          errorMessage: 'I am the cause',
+          stacktrace: expect.arrayContaining([
+            expect.objectContaining({
+              file: expect.any(String),
+              method: expect.any(String),
+              lineNumber: expect.any(Number),
+              columnNumber: expect.any(Number)
+            })
+          ]),
+          type: 'browserjs'
+        }))
+    })
+
+    it('converts a string cause into an exception', () => {
+      const err = new Error('I am the error')
+      // @ts-ignore
+      err.cause = 'I am not a real cause'
+      // @ts-ignore
+      const event = Event.create(err, true, undefined, 'notify()', 0)
+      expect(event.errors.length).toBe(2)
+      expect(event.errors).toContainEqual({
+        errorClass: 'Error',
+        errorMessage: 'I am not a real cause',
+        stacktrace: [],
+        type: 'browserjs'
+      })
+    })
+
+    it('handles invalid cause errors', () => {
+      const err = new Error('I am the error')
+      // @ts-ignore
+      err.cause = { error: 'I am not a real cause' }
+      // @ts-ignore
+      const event = Event.create(err, true, undefined, '', 0)
+      expect(event.errors.length).toBe(2)
+      expect(event.errors).toContainEqual({
+        errorClass: 'InvalidError',
+        errorMessage: 'error cause was a non-error. See "error cause" tab for more detail.',
+        stacktrace: [],
+        type: 'browserjs'
+      })
+      expect(event.getMetadata('error cause')).toEqual({ error: 'I am not a real cause' })
+    })
+
+    it('tolerates non-errors regardless of tolerateNonErrors being true/false', () => {
+      const err = new Error('I am the error')
+      // @ts-ignore
+      err.cause = 'I am not a real cause'
```

Comment:
Probably wouldn't have mentioned / noticed this if the other discussions hadn't drawn attention to it, but it's not really true that it's not a cause... The key thing is it's not an `Error` 😄 
```suggestion
      err.cause = 'I am not an Error'
```